queries.sql

```diff
@@ -45,7 +45,20 @@ SELECT SUM(escape_attempts) AS escape_attempts, neutered FROM animals GROUP BY n
 SELECT MAX(weight_kg), MIN(weight_kg), species FROM animals GROUP BY species;
 SELECT AVG(escape_attempts) AS avg_escape_attempts, species FROM animals  WHERE date_of_birth BETWEEN '1990/01/01' AND '2020/12/31' GROUP BY species;
 
-
+--What animals belong to Melody Pond?
+SELECT * FROM owners JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
```

Comment:
- [ ] Here you have written incorrectly we have to show `which animals belong to Melody Pond?`
```
SELECT * FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   WHERE full_name = 'Melody Pond';
```